bertytech/content/blog/bluetooth-low-energy/index.md

```diff
@@ -26,7 +26,7 @@ Joking aside, to celebrate the fact that our first transport allowing direct con
 
 ![Back to the Future gif](rkbQMNjvI.gif)
 
-At that time, after about a month of research and testing, the team agreed on implementing the following stack for our chat application:
+At that time, after about a month of research split in two team, one team testing Peer-to-Peer networking technologies, and another one experimenting with direct transport technologies - more precisely Apple MultipeerConnectivity - we agreed on implementing the following stack for our chat application:
```

Comment:
At that time, after about a month of research split in two teams, one team testing Peer-to-Peer networking technologies, and another one experimenting with direct transport technologies - more precisely Apple MultipeerConnectivity - we agreed on implementing the following stack for our chat application: